cmd/swarmd/main.go

```diff
@@ -55,7 +55,6 @@ var (
 			}
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			ctx := context.Background()
```

Comment:
@stevvooe, I think this is the right fix - `context.Background()` is only used once later to create the actual `ctx` used throughout the function:

`ctx, cancel := context.WithCancel(context.Background())`